exp/lighthorizon/actions/helpers_test.go

```diff
@@ -0,0 +1,33 @@
+package actions
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/go-chi/chi"
+)
+
+func makeRequest(
+	t *testing.T,
+	queryParams map[string]string,
+	routeParams map[string]string,
+) *http.Request {
+	request, err := http.NewRequest("GET", "/", nil)
+	if err != nil {
+		t.Fatalf("unexpected error %v", err)
+	}
```

Comment:
might as well use `require` here for brevity
```suggestion
	request, err := http.NewRequest("GET", "/", nil)
	require.NoError(t, err)
```